### PR TITLE
Epa model update

### PR DESCRIPTION
**Pull request by saiemgilani**
the main epa_wpa portion of the cfb_pbp_data function is redesigned to look like this:
```    
g_ids = sort(unique(play_df$game_id))
play_df = purrr::map_dfr(g_ids,
                         function(x) {
                           play_df %>%
                             dplyr::filter(.data$game_id == x) %>%
                             penalty_detection() %>% 
                             add_play_counts() %>% 
                             clean_pbp_dat() %>% 
                             clean_drive_dat() %>% 
                             prep_epa_df_after() %>% 
                             create_epa() %>%
                           # add_betting_cols(g_id = x, yr=year) %>%
                           # create_wpa_betting() %>%
                            create_wpa_naive()
                        })
```

Will edit to complete this PR

### CODE REVIEW

File: R/cfb_betting_lines.R [hunk 1]
```diff
@@ -157,10 +158,10 @@ cfb_betting_lines <- function(game_id = NULL,
           dplyr::rename(game_id = .data$id) %>% 
           as.data.frame()
       }
-      message(glue::glue("{Sys.time()}: Scraping betting lines data..."))
+      # message(glue::glue("{Sys.time()}: Scraping betting lines data..."))
```
Comment (meysubb) [inline R/cfb_betting_lines.R:161]: Are we ever going to bring this back?

Comment (meysubb) [reply R/cfb_betting_lines.R:161]: If not, I'd just say delete it.

File: R/create_wpa_naive.R [hunk 2]
```diff
@@ -87,9 +89,25 @@ create_wpa_naive <- function(df, wp_model = cfbscrapR:::wp_model) {
   df = df %>% 
     dplyr::arrange(.data$game_id, .data$new_id)
   
+  
+  
   Off_Win_Prob = as.vector(predict(wp_model, newdata = df, type = "response"))
   df$wp_before = Off_Win_Prob
-
+  # Kickoff plays
+  # Calculate EP before at kickoff as what happens if it was a touchback
```
Comment (meysubb) [inline R/create_wpa_naive.R:97]: This is for WP not EP right? Can we update comment. I know it was a copy/paste thing

Comment (meysubb) [reply R/create_wpa_naive.R:97]: So WP on kickoffs is compared relative to a touchback?

Comment (saiemgilani) [reply R/create_wpa_naive.R:97]: Yep, just a copy paste non-change

File: R/cfb_betting_lines.R (hunk 1 above)
Comment (saiemgilani) [reply R/cfb_betting_lines.R:161]: On this one, probably not. I leave the error in there for now, but I was trying to think of any clever ways to message this that aren't overly verbose. I can delete

**Pull request merged**